src/WallGo/equationOfMotion.py

```diff
@@ -828,6 +833,14 @@ def wallPressure(
             Tplus,
             Tminus,
         )
+        temperatureProfile = None
+        velocityProfile = None
+        if not self.forceEnergyConservation:
```

Comment:
I'm a bit confused. What determines these velocity and temperature profiles now?

Reply:
If we don't enforce conservation, we use the profiles computed at the first iteration. Otherwise, we update the profiles at each iteration.

Reply:
But why would those be the correct ones? Or is the point that they 'drop out'?